### ISSUE TITLE
Fonts always regenerate modified files

### ISSUE THREAD

**Issue by mdo** [tancredi/fantasticon#90]
We're using Fantasticon in Bootstrap Icons and it's been awesome, thanks so much for maintaining it!

One issue I've run into is that font files will always regenerate with changes to them in our repo, even when there are no new SVGs or changes to existing SVGs. This feels like it could be a bug in some way, or maybe there's a way to prevent this from happening that we haven't found in the project.

X-ref: https://github.com/twbs/icons/issues/618

/cc @XhmikosR

**Comment by XhmikosR** [comment]
To reproduce:

```
git clone https://github.com/twbs/icons.git
npm i
npm run icons-font
```

You will see that the font files are modified while no SVG files have changed. I tried to have a look but I'm not totally familiar with the codebase @tancredi. It might be something to do with svg2woff or something.

**Comment by tancredi** [comment]
Hey @mdo @XhmikosR - the TTF generator accepts a timestamp option used for meta-data (or similar) which defaults to current time which was making the generated output unpredictable - I'm now passing `0` by default to make the build reliable.

The change is available in `v1.1.0` - please let me know whether there's any more issues with this.

Also, very happy it's being used by Bootstrap Icons 🙂🙂 looks really cool 🔥

**Comment by XhmikosR** [comment]
Confirmed, thanks!
